web/tsconfig.json

```diff
@@ -10,5 +10,8 @@
     "noUnusedLocals": true,
     "sourceMap": true
   },
-  "include": ["src"]
-}
+  "include": [
+    "src",
+    "jest.config.js"
+  ]
+}
```

Comment:
```suggestion
}

```

Reply:
Added new line at the end of the file in most recent commit.